frontend/src/metabase/query_builder/components/QuestionEmbedWidget/QuestionEmbedWidget.tsx

```diff
@@ -0,0 +1,66 @@
+import type { Card } from "metabase-types/api";
+import type { EmbedOptions } from "metabase-types/store";
+import type { ExportFormatType } from "metabase/dashboard/components/PublicLinkPopover/PublicLinkCopyPanel";
+import { useDispatch, useSelector } from "metabase/lib/redux";
+import * as Urls from "metabase/lib/urls";
+import { EmbedModal } from "metabase/public/components/widgets/EmbedModal";
+import EmbedModalContent from "metabase/public/components/widgets/EmbedModalContent";
+import { getMetadata } from "metabase/selectors/metadata";
+import { getCardUiParameters } from "metabase-lib/parameters/utils/cards";
+
+import {
+  createPublicLink,
+  deletePublicLink,
+  updateEnableEmbedding,
+  updateEmbeddingParams,
+} from "../../actions";
+
+type QuestionEmbedWidgetProps = {
+  className?: string;
+  card: Card;
+  onClose: () => void;
+};
+export const QuestionEmbedWidget = (props: QuestionEmbedWidgetProps) => {
+  const { className, card, onClose } = props;
+
+  const metadata = useSelector(getMetadata);
+
+  const dispatch = useDispatch();
+  const createPublicQuestionLink = () => dispatch(createPublicLink(card));
+  const disablePublicQuestionLink = () => dispatch(deletePublicLink(card));
```

Comment:
Ah! I remember now - SharingPane used the `disablePublicLink` naming so I went with that to make my life easier. I'll change all of those because it will get confusing